src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/info-group/info-group.component.scss

```
@@ -1,4 +1,14 @@
+@use './src/styles/defaults/bootstrap-defaults' as bd;
+
 .info-group-title {
   font-size: 1.75rem;
   margin: 0 0 0.5vw 0.5vw;
 }
+
+.popover-icon {
+  border: 0;
+  font-size: 1.2rem;
+  margin-left: 2px;
+  outline-style: none;
+  padding: 0;
+}
```

With the usage of `btn btn-link p-0` on the button, you only need to following to get a working button:
```suggestion
.popover-icon:focus {
  box-shadow: none;
}
```